plugins/vli/fu-vli-common.c

```diff
@@ -57,37 +57,39 @@ fu_vli_common_device_kind_get_size(FuVliDeviceKind device_kind)
 	if (device_kind == FU_VLI_DEVICE_KIND_VL813)
 		return 0x8000;
 	if (device_kind == FU_VLI_DEVICE_KIND_VL815)
-		return 0x20000 * 2;
+		return 0x10000;
 	if (device_kind == FU_VLI_DEVICE_KIND_VL817)
-		return 0x20000 * 2;
+		return 0x10000;
 	if (device_kind == FU_VLI_DEVICE_KIND_VL817S)
-		return 0x20000 * 2;
+		return 0x10000;
 	if (device_kind == FU_VLI_DEVICE_KIND_VL819Q7)
-		return 0x20000 * 2;
+		return 0x10000;
```

Comment:
woah, that seems dramatic?

Reply:
previous code were incorrect. Our code size is only 0x10000.